ci/testrmsenv.sh

```diff
@@ -0,0 +1,51 @@
+# This shell script is to be sourced and run from a github workflow
+# when xtgeo is to be tested towards a new RMS Python enviroment
+
+run_tests() {
+    set_test_variables
+
+    copy_test_files
+
+    install_test_dependencies
+
+    run_pytest
+}
+
+set_test_variables() {
+    echo "Setting variables for xtgeo tests..."
+    CI_TEST_ROOT=$CI_ROOT/xtgeo_test_root
+}
+
+copy_test_files () {
+    echo "Copy xtgeo testdata..."
+    if ! [ -d $XTGEO_TESTDATA_PATH ]; then
+        echo "Cloning xtgeo-testdata to $XTGEO_TESTDATA_PATH..."
+        git clone --depth=1 https://github.com/equinor/xtgeo-testdata $XTGEO_TESTDATA_PATH
```

Comment:
Might be good to cache this locally (maybe you have it planned?) since it's a few GB. But I guess this variable can just point to that cache too if set before